docs/examples/viz_parametric_surfaces_v2.py

```diff
@@ -0,0 +1,250 @@
+###############################################################################
+# Importing necessary modules
+
+from fury import window, actor, ui
+import numpy as np
+from scipy.spatial import Delaunay
+
+###############################################################################
+# Function that returns the parametric surface specified by the user.
+
+
+def parametric_func(name, colors=(1, 1, 1), coords=[0, 0, 0], scale=1.0):
+    """
+    Parameters
+    ----------
+    name : string - "mobius_strip" or "kleins_bottle" or "roman_surface"
+        or "boys_surface" or "bohemian_dome" or "dinis_surface" or
+        "pluckers_conoid" or "bours_surface". Determines the parametric surface
+        to be returned.
+    colors : ndarray (3) or tuple (3)
+        Determines the color of the surface. (default : white)
+    coords : ndarray (3)
+        Determines the coordinates where the parametric surface will be
+        rendered. (default : [0, 0, 0])
+    opacity : float, optional
+        Takes values from 0 (fully transparent) to 1 (opaque). (default : 1)
+    scale : float, optional
+        Determines the size of the parametric surface returned. (default : 1)
+    cmap : bool, optional
+        Set it to true if colormap is being mapped to the surface.
+    cmap_name : string, optional
+        Stores the name of the colormap specified by the user, cmap should be
+        set to True for the colormap to be rendered.
+
+    Returns
+    -------
+    vtkActor
+
+    """
```

Comment:
There are three files in the example and at least two different techniques working (Delaunay triangulation and point-based). It would be great to merge these different techniques to render the curves into a single function and have a parameter to select between them.